### PR TITLE
Add FAQs for Expectation Range and Industry Benchmark

### PR DESCRIPTION
**Pull request by rudibatt**
The English text was originally written by me and reviewed and optimized by @jonathanpross and Andi.
German and Swabian in Rudi Freestyle
French by G-Translate

### CODE REVIEW

File: ui/readme.swg.md [hunk 1]
```diff
@@ -184,6 +184,30 @@ Dr Tabelle "Suchbegriffe mit niedriger CR" isch no extra gfiltert für Begriffe
 
 Dr Chart "Entwicklung Suchen ohne Ergebnis" zeigt, wieviel Anteil von all d'Suche (net Begriffe) ohne Treffer an all Suchanfragen send.
 
+Älle Diagramme werda im Konteschd von noch mehr Data a'zeigt. Dodafür gibts folgende Möglichkeita:
```
Comment (HJK181) [inline ui/readme.swg.md:187]: The Swebian readme currently only contains English text. Please paste the English version here or start translating the complete file :)

File: ui/readme.de.md [hunk 2]
```diff
@@ -176,6 +176,29 @@ Die Tabelle "Suchbegriffe mit niedriger CR" ist ergänzend dazu gefiltert auf Su
...
+
+### Industry Benchmark
+Der 'Industry Benchmark' ist ein Vergleich mit normalisierten Durchschnittswerten anderer Kunden.
+
+*Zweck*:
+Es gibt viele Faktoren die bestimmen, ob ein KPI-Wert gut ist oder nicht. Auf basis statistisch relevanter Daten anderer Kunden, können wir einen hochwertigen Vergleich in Form des 'Industry Benchmark' liefern.
+Damit werden die Werte eines Diagrams in einen erweiterten Kontext eingebettet und geben damit Einsicht in die Qualität Ihrer Suche. Es gibt Einsicht über das Optimierungs-Potential und zeigt auf, ob sich Trends und saisonale Effekte auch in Ihrem Shop zeigen.
+
+*Wie es berechnet wird*:
+Wir verwenden gefilterte und aggregierte Werte verschiedener repräsentativer Date aus unterschiedlichen Domänen um eine allgemeine Wahrscheinlichkeitsverteilung abzubilden. Auf Basis dessen wird der 95%ige Konfidenzinterval für jeden Tageswert gebildet.
```
Comment (HJK181) [inline ui/readme.de.md:200]: Date**n**

**Review by cgicgi — APPROVED** [review]
I have committed my changes directly. I especially removed the reference to "customer" and replaced it with "retailers" - we shouldn't communicate this so openly

File: ui/readme.swg.md (hunk 1 above)
Comment (rudibatt) [reply ui/readme.swg.md:187]: Oh, ok. There are already other parts that are in Swäbian. And I think English is the least option. I will create another PR that replaces the English Part with the German one.

Comment (HJK181) [reply ui/readme.swg.md:187]: That's strange as I recently asked Siggi to replace the mixed text with complete English content, and he did ...

**Pull request merged**